### PR TITLE
Create software-as-a-service-saas-tracking-pixel.md

### PR DESCRIPTION
**Pull request by rpdubose**

### CODE REVIEW

**Comment by rpdubose** [comment]
@srveit Don't merge until you hear from me.

**Comment by rpdubose** [comment]
@srveit I feel like the checks are failing b/c they are all new files. Can you take a look?

File: Marketplace/Providers/usage-based-billing.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{{{
+ "title": "Usage Base Billing",
+ "date": "04-02-2017",
+ "author": "Rich DuBose",
+ "attachments": [],
+ "contentIsHTML": false
+ +}}}
```
Comment (srveit) [inline Marketplace/Providers/usage-based-billing.md]: Change `+}}}` to `}}}`

File: Marketplace/Providers/usage-based-billing.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+ 
+ ###Usage Based Billing
+ 
+ Many providers require a usage based element to their subscription model. The CenturyLink Cloud Marketplace does support **Usage Based Billing** via our [v2 SaaS API](/software-as-a-service-saas-v2-api.md).
```
Comment (srveit) [inline Marketplace/Providers/usage-based-billing.md]: Change link to `./software-as-a-service-saas-v2-api.md`

File: Marketplace/Providers/software-as-a-service-saas-v1-api.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+"title": "Software as a Service (SaaS) v1 API",
+"date": "04-02-2017",
+"author": "Rich DuBose",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+### Overview
+
+*New integrations with the v1 API are not permitted after April 8, 2017. All providers should should reference the [SaaS v2 API documentation](/software-as-a-service-saas-v2-api.md)*
```
Comment (srveit) [inline Marketplace/Providers/software-as-a-service-saas-v1-api.md]: Change link to `./software-as-a-service-saas-v2-api.md`

File: Marketplace/Providers/usage-based-billing.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+ "title": "Usage Base Billing",
+ "date": "04-02-2017",
+ "author": "Rich DuBose",
+ "attachments": [],
+ "contentIsHTML": false
+ }}}
+ 
+ ###Usage Based Billing
+ 
+ Many providers require a usage based element to their subscription model. The CenturyLink Cloud Marketplace does support **Usage Based Billing** via our [v2 SaaS API](/software-as-a-service-saas-v2-api.md).
```
Comment (srveit) [inline Marketplace/Providers/usage-based-billing.md]: Change link to `./software-as-a-service-saas-v2-api.md`

**Pull request merged**